tests/integration/schema_registry/data/Makefile

```diff
@@ -0,0 +1,16 @@
+WORK_DIR:=$(strip $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
```

Comment:
realpath is a GNU extension which might not be generally available.
See if this works instead:
```makefile
WORK_DIR := $(dir $(lastword $(MAKEFILE_LIST)))
```